pillar/redmine.sls

```diff
@@ -0,0 +1,42 @@
+network:
+  host_id: ocp16
+  ipv4: 139.162.219.246
+#  ipv6: 2a01:7e00::f03c:93ff:fe24:07b9
+#  netplan:
+#    template: linode
+#    addresses:
+#      - 2a01:7e00::f03c:93ff:fe24:07b9/64 # SLAAC
+#    gateway4: 198.51.100.1
+#    gateway6: fe80::1
+
+apache:
+  public_access: True
+  sites:
+    crm:
+      configuration: redmine
+      servername: crm.open-contracting.org
+
+mysql:
+  version: 8.0
+  configuration: False
+  databases:
+    redmine:
+      user: redmine
```

Comment:
We usually put this in the private file where the user is created (with password), to avoid splitting the user's name across two files.

Reply:
Moved the MySQL DB configuration next to the user configuration in pillar private.